Boa.Constrictor/Logging/Models/ArtifactTypes.cs

```diff
@@ -20,5 +20,15 @@ public class ArtifactTypes
         /// Artifact type for screenshot images.
         /// </summary>
         public const string Screenshots = "Screenshots";
```

Comment:
Would it be better for this to be an enum? Or are you wanting it to be easy for people to add new Artifact Types?